src/Main/Main.java

```diff
@@ -1,7 +1,12 @@
 package Main;
 
+import PasswordStorage.ManagePassword;
+
 public class Main {
     public static void main(String[] args) {
-        
+
```

Comment:
Should remove all changes from Main, make a test file for running your stuff or don't push your changes to the main class